api/database/factories/ModelFactory.php

```diff
@@ -106,3 +107,20 @@
 
     return compact('token') + $body;
 });
+
+$factory->define(App\Models\ArticlePermalink::class, function ($faker) {
+    return [
+        'permalink' => str_random(255),
```

Comment:
my issues where somehow connected with unique syntax, just was not much into debugging the issue at that time
will debug it now